index.html

```diff
@@ -64,5 +65,192 @@ <h2><a href="https://www.linkedin.com/in/ivan-martinez-von-halle/" class="connec
+          </ul>
+          <button id="pr-bt-4" class="project-btn">See Project</button>
+        </div>
+      </div>
+   </section>
+   <!-- works section (end) -->
+
+ <!-- about section (init)-->
+<section id="about-sec">
+  <div class="about-me">
+    <h2 id="tit-ab-me" class="title">About Myself</h2>   
+    <p id="intro-text" class="parrafo">
+      Hello, I'm a software developer! I can help you build a product,
+      feature or website Look through some of my work and experience!
+      If you like what you see and have a project you need coded, don't hesitate to contact me.
+    </p>   
+    <h3><a href="https://linkedin.com/in/ivan-martinez-von-halle/" class="connect-text">LET'S CONNECT</a></h3>
+    <ul class="social-media-icons">
+      <li>
+        <a href="https://twitter.com/imprivado" aria-label="Twitter">
+          <img src="./images/first-page-img/twitter.svg" alt="twitter logo">
+        </a>
+      </li>
+      <li>
+        <a href="https://linkedin.com/in/ivan-martinez-von-halle/" aria-label="LinkedIn">
+          <img src="./images/first-page-img/linkedin.svg" alt="linkedin logo">
+        </a>
+      </li>
+      <li>
+        <a href="" aria-label="Medium">
+          <img src="./images/first-page-img/medium.svg" alt="medium logo">
+        </a>
+      </li>
+      <li>
+        <a href="https://github.com/ivanmvh" aria-label="GitHub">
+          <img src="./images/first-page-img/github.svg" alt="github logo">
+        </a>
+      </li>
+      <li>
+        <a href="" aria-label="Angel list">
+          <img src="./images/first-page-img/peace.svg" alt="Angel list logo">
+        </a>
+      </li>
+    </ul>
+    <button class="resume-btn">Get my resume</button>
+  </div>
+
+  <div class="about-languages">
+    <div class="languages-section">
+      <h3 class="languages">Languages</h3>
+      <img class="arrow" src="./images/languages-img/down-arrow.svg" alt="down-arrow" height="15" width="15">
+    </div>
+    <div class="languages-boxes">
+      <div class="language-box">
+        <img src="./images/languages-img/JS.svg" alt="javascript image" height="50" width="50">
+        <p>JavaScript</p>
+      </div>
+      <div class="language-box">
+        <img src="./images/languages-img/html.svg" alt="html image" height="50" width="50">
+        <p>HTML</p>
+      </div>
+      <div class="language-box">
+        <img src="./images/languages-img/css.svg" alt="css image" height="50" width="50">
+        <p>CSS</p>
+      </div>
+    </div>
+    <div class="languages-section frameworks">
+      <h3 class="languages">Frameworks</h3>
+      <img class="arrow" src="./images/languages-img/right-arrow.svg" alt="right-arrow" height="15" width="15">
+    </div>
+    <div class="languages-section skills">
+      <h3 class="languages">Skills</h3>
+      <img class="arrow" src="./images/languages-img/right-arrow.svg" alt="right-arrow" height="15" width="15">
+    </div>
+  </div>
```

Comment:
I suggest you change this into `<ul>` `<li>` tags. This is so that you can match semantic requirement as well as it will be easier if you want to put them as links